app/lib/src/pages/full_error_page.dart

```diff
@@ -1,7 +1,9 @@
+import 'package:auto_route/auto_route.dart';
 import 'package:flutter/material.dart';
 import 'package:mondaymorning/src/services/themes/index.dart';
 import 'package:mondaymorning/src/ui/screens/full_error_screen.dart';
 
+@RoutePage<dynamic>(name: 'NotFoundRoute')
```

Comment:
Why dynamic? This page doesn't return any data so it should be void instead.

```suggestion
@RoutePage<void>(name: 'NotFoundRoute')
```